### ISSUE TITLE
Sentry.startTransaction is not a function

### ISSUE THREAD

**Issue by not-the-car** [getsentry/sentry-electron#352]
## Versions + Platform

- [ 2.5.1] SDK version - `@sentry/electron@v?.?.?`
- [12.0.2 ] Electron version - `electron@v?.?.?`
- [ Windows] Platform - `Windows/macOS/Linux`

## Description

The initialized Sentry instance does not contain a `startTransaction` method, despite the [documentation stating so](https://docs.sentry.io/platforms/javascript/guides/electron/configuration/sampling/):

```ts
Sentry.startTransaction(
  {
    // `transactionContext` - will be recorded on transaction
    name: 'Search from navbar',
    op: 'search',
    tags: {
      testGroup: 'A3',
      treatmentName: 'eager load',
    },
  },
  // `customSamplingContext` - won't be recorded
  {
    // PII
    userId: '12312012',
    // too big to send
    resultsFromLastSearch: { ... }
  },
);
```

**Comment by umerkk** [comment]
+1 - Much needed

**Comment by not-the-car** [comment]
Certainly don't want to just pile on - the libraries Sentry provides are great for the most part. But the electron sdk needs work. The typescript support is not there (initialization/configuration options are not correct) either. I ended up just going with node and react sentry libraries.

**Comment by umerkk** [comment]
I wonder, is there a way to roam hubs around multiple sdks.
We have the same problem, using electron sdk but realized the native node & react sdks provides much rich interfaces that electron does not.
But I do like Electron version of crash dumps that I am certainly gonna miss if I opt for Node, React library in my election app.

**Comment by not-the-car** [comment]
Agreed. I'm sure there are ways, but just don't have the bandwidth to investigate currently.

**Comment by timfish** [comment]
You can use all the Sentry React components with sentry-electron.

**Comment by timfish** [comment]
> The typescript support is not there (initialization/configuration options are not correct) either.

@not-the-car can you open an issue for these? I'm not aware of any issues with the TypeScript support

**Comment by not-the-car** [comment]
@timfish double checked on a fresh repo and all seems to be well, sorry for the inaccurate report!

**Comment by umerkk** [comment]
@timfish - How does it work? specifically regarding the initialization of these SDKs.
Currently, we are initializing renderer electronSDK in. Preload, does that mean we also have to init React SDK? How does the Hub & Scope sharing work?

Also, does that mean we can use NodeSDK in the backend for APM capabilities since Electron SDK doesn't provide that?

> You can use all the Sentry React components with sentry-electron.

**Comment by timfish** [comment]
The React SDK is just a thin wrapper around the Browser SDK. The Electron SDK is just a wrapper around the Browser/Node SDKs.

https://github.com/getsentry/sentry-electron/issues/295#issuecomment-849158452
